guides/common/modules/proc_viewing-vulnerability.adoc

```diff
@@ -0,0 +1,22 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="viewing-vulnerability"]
+= Viewing vulnerability
```

Comment:
I'm wondering if this could have a better title. What about _Examinig vulnerability of systems_?

Reply:
Two things:

1. I'm going with singular vulnerability.
2. The web UI doesn't contain the "status" keyword. I would avoid it so that it doesn't confuse users.